Gemfile

```diff
@@ -24,5 +24,6 @@ group :development, :test do
 end
 
 group :development do
+  gem "dockerfile-rails", ">= 1.2"
```

Comment:
If you have generated this image why you still needs this gem here? 🤔.

Reply:
great question 👀